specifyweb/frontend/js_src/lib/components/TreeView/Tree.tsx

```diff
@@ -106,6 +108,28 @@ export function Tree<
     [baseUrl, statsThreshold]
   );
 
+  const treeDefinition = treeDefinitionItems[0].treeDef;
+  const treeDefId = idFromUrl(treeDefinition);
+  const createRootNode = async (): Promise<void> => {
+    if (treeDefId === undefined) {
+      console.error('treeDefId is undefined');
```

Comment:
(optional) consider using softError instead of console.error
it is equivalent to console.error in production, but will produce a full-screen exception in development.
that is useful as it makes it less likely that bugs get unnoticed in development, while preventing non-critical errors from disrupting user's work